corehq/apps/app_manager/static/app_manager/js/download_index_main.js

```diff
@@ -0,0 +1,33 @@
+
+hqDefine('app_manager/js/download_index_main',[
+   'jquery',
+   'ace-builds/src-min-noconflict/ace',
+    'app_manager/js/download_async_modal',
+    'app_manager/js/source_files'
+],function ($) {
```

Comment:
`ace` should be added to params here since it's accessed below

Reply:
ohh shit. Stupid mistake :triumph: . I should have been more careful.